website/www/site/content/en/documentation/io/developing-io-java.md

```diff
@@ -17,6 +17,9 @@ limitations under the License.
 -->
 # Developing I/O connectors for Java
 
+**IMPORTANT:** Please use ``Splittable DoFn`` to develop your new I/O. For more details, please read
```

Comment:
Remove instances of "please" on this page

Reference: https://developers.google.com/style/tone#politeness